### PR TITLE
Further type refactoring

### PR DESCRIPTION
**Pull request by tommyrharper**

### CODE REVIEW

File: markets/perps-market/contracts/interfaces/IPerpsAccountModule.sol [hunk 1]
```diff
@@ -73,7 +73,7 @@ interface IPerpsAccountModule {
      * @param accountId Id of the account.
      * @return openInterestValue total open interest value of the account.
      */
-    function totalAccountOpenInterest(uint128 accountId) external view returns (uint256);
+    function totalAccountOpenInterest(uint128 accountId) external view returns (USDUint256);
```
Comment (JaredBorders) [inline markets/perps-market/contracts/interfaces/IPerpsAccountModule.sol:76]: hmm. So, from docs: "Market Skew refers to the difference in open interest between long and short positions in the market"

And we define skew in `BaseQuantoPerUSDUint256` units. So wouldn't the difference in skew be in `BaseQuantoPerUSDUint256` units too?

File: markets/perps-market/contracts/modules/PerpsAccountModule.sol [hunk 2]
```diff
@@ -96,7 +96,7 @@ contract PerpsAccountModule is IPerpsAccountModule {
     /**
      * @inheritdoc IPerpsAccountModule
      */
-    function totalAccountOpenInterest(uint128 accountId) external view override returns (uint256) {
+    function totalAccountOpenInterest(uint128 accountId) external view override returns (USDUint256) {
```
Comment (JaredBorders) [inline markets/perps-market/contracts/modules/PerpsAccountModule.sol:99]: same q here; why not `BaseQuantoPerUSDUint256`?

Comment (JaredBorders) [reply markets/perps-market/contracts/modules/PerpsAccountModule.sol:99]: is it just how it must be reported to the market? it is external so I don't see why that would be it...

**Review by JaredBorders — APPROVED** [review]
Approved, but let's talk about the OI question during sync.

The code looks great! New `greaterThanZero()` is 🧼 (clean) 

ty @Flocqst

File: markets/perps-market/contracts/modules/PerpsAccountModule.sol (hunk 2 above)
Comment (tommyrharper) [reply markets/perps-market/contracts/modules/PerpsAccountModule.sol:99]: added docs explaining this

File: markets/perps-market/contracts/interfaces/IPerpsAccountModule.sol (hunk 1 above)
Comment (tommyrharper) [reply markets/perps-market/contracts/interfaces/IPerpsAccountModule.sol:77]: added docs explaining this

**Pull request merged**